packages/core/src/index.ts

```diff
@@ -304,19 +277,31 @@ export const removeSessionProperty = (key: string) => {
 };
 
 export const getSessionProperties = () => {
-  return NativeModules.EmbraceManager.getSessionProperties();
+  // TODO REFACTOR WHEN iOS IMPLEMENT THE METHOD
+
+  // return NativeModules.EmbraceManager.getSessionProperties();
+
+  return createFalsePromise();
 };
 
 export const endSession = (clearUserInfo: boolean = false) => {
-  return NativeModules.EmbraceManager.endSession(clearUserInfo);
+  return NativeModules.EmbraceManager.endSession();
```

Comment:
we need to continue to pass `clearUserInfo` through here since the Android native module expects it, we'll just have to ignore on the iOS side

Reply:
mmm in my local I dont have that parameter, maybe this PR is not up to date

Anyway I will push right now the test fix

I removed the parameter `clearUserInfo: boolean = false`. since android expose 2 methods, and I will use the first method, without parameters to have the same method signature and functionality than iOS

Android SDK
```
    @Override
    public synchronized void endSession() {
        endSession(false);
    }
```